config/initializers/safemode_jail.rb

```diff
@@ -19,3 +19,7 @@ class ActiveRecord::Batches::BatchEnumerator::Jail < Safemode::Jail
 class URI::Generic::Jail < Safemode::Jail
   allow :host, :path, :port, :query, :scheme
 end
+
+class ActiveSupport::TimeWithZone::Jail < Safemode::Jail
+  allow *(Safemode.core_jail_methods(Time).uniq)
```

Comment:
It looks like it will allow Ruby's `Time`'s methods defined here [1], which should work if I understand right the compatibility [2] between the two, but it won't allow actual `TimeWithZone` methods. If this PR is about to allow work with `TimeWithZone` objects as with `Time` objects, then it's :+1: from me and nothing here to add. Although it would nice to have the documentation for allowed methods in case users are not aware they can find it in the official Rails docs...

[1] - https://github.com/svenfuchs/safemode/blob/9b358063fba33b862cf104d341d0800eca0ad6c2/lib/safemode/core_jails.rb#L101-L107
[2] - https://github.com/rails/rails/blob/83217025a171593547d1268651b446d3533e2019/activesupport/lib/active_support/time_with_zone.rb#L49

Reply:
`allow *(Safemode.core_jail_methods(Time).uniq)` will enable quite a lot of methods, do we actually need all of them?

```ruby
Safemode.core_jail_methods(Time).uniq
=> ["blank?",
 "_dump",
 "asctime",
 "ctime",
 "day",
 "dst?",
 "getgm",
 "getlocal",
 "getutc",
 "gmt?",
 "gmt_offset",
 "gmtime",
 "gmtoff",
 "hash",
 "hour",
 "httpdate",
 "isdst",
 "iso8601",
 "localtime",
 "mday",
 "min",
 "minus_without_duration",
 "mon",
 "month",
 "plus_without_duration",
 "present?",
 "rfc2822",
 "rfc822",
 "sec",
 "strftime",
 "succ",
 "to_date",
 "to_datetime",
 "to_f",
 "to_i",
 "tv_sec",
 "tv_usec",
 "usec",
 "utc",
 "utc?",
 "utc_offset",
 "wday",
 "xmlschema",
 "yday",
 "year",
 "zone",
 "to_formatted_s",
 "+",
 "-",
 "<",
 "<=",
 "<=>",
 "!",
 "!=",
 "==",
 "===",
 ">",
 ">=",
 "eql?",
 "equal?",
 "methods",
 "is_a?",
 "kind_of?",
 "nil?",
 "to_a",
 "to_jail",
 "to_s",
 "inspect",
 "to_param",
 "freeze"]
```

Reply:
They all seemed to be allowed by Safemode itself anyway, so why not? Moreover, you can find those methods in our doc in  the template editor's page (`Help` section). It'll simply allow all the default (already allowed) methods of `Time` for `TimeWithZone`. One thing to concider though: we are not adding any new (probably more useful) method from actual `TimeWithZone` class.